sql/catalyst/benchmarks/EnumTypeSetBenchmark-results.txt

```diff
@@ -1,104 +1,105 @@
-Java HotSpot(TM) 64-Bit Server VM 1.8.0_152-b16 on Linux 4.14.0_1-0-0-42
-Intel(R) Xeon(R) Gold 6271C CPU @ 2.60GHz
+OpenJDK 64-Bit Server VM 1.8.0_312-b07 on Linux 5.11.0-1022-azure
+Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
 Test contains use empty Set:              Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-Use HashSet                                           1              1           0       1316.2           0.8       1.0X
-Use EnumSet                                           0              0           0   16393442.6           0.0   12454.9X
+Use HashSet                                           0              1           1       2192.0           0.5       1.0X
+Use EnumSet                                           0              0           0     Infinity           0.0  InfinityX
```

Comment:
This benchmark looks a little weird although the previous ratio is also huge already.